Coucat/Pokedex
# Header design

# Pull Request

## Description

### Ticket impemented #[POK-17](https://coucat.atlassian.net/browse/POK-17)

[//]:<> (You can see the number of ticket in Jira in the top of task, replace the ticketNumber in the space and the number in the link)
### What did you implement it?

[//]:<> (Mark with an 'x' or a space)

- [ ] Bug Fix (Non-breaking change wich fixes an issue)
- [x] New Feature (Non-breaking change wich adds functionality)


### How did you implement it?
[//]:<> (In this section describe all things that you made)
The header was made with figma design. 
Implementing styled components with react to do all the design.
Sections of header have a hover animation and have active class to remark when is focus.
Header is responsive with a min-width to small displays. 
## How can we test this ticket?
[//]:<> (Write in this part all the commands that you can use to test your code)
Active class is already in the page(is the red one) and to see the hover animation only needs mouse over the section.
To test the min-width you can inspect the page and push __ctrl + shift + M__
## Checklist
[//]:<> (Mark with a 'x' or a space )
